Tests/js/views/services/plugins/assets/ez-searchplugin-tests.js

```diff
@@ -111,7 +147,8 @@ YUI.add('ez-searchplugin-tests', function (Y) {
             this.view.fire('contentSearch', {
                 viewName: this.viewName,
                 search: {
-                    criteria: {},
+                    offset: 42,
```

Comment:
Well no, offset and limit aren't mandatory... But the query mocks now expect 2 args number now for their setLimitAndOffset method (that's why i had Y.Mock.Any before). So now I had to fill the tests with limit and offset.